artemis-journal/src/main/java/org/apache/activemq/artemis/core/io/AbstractSequentialFile.java

```diff
@@ -124,6 +125,9 @@ public void copyTo(SequentialFile newFileName) throws Exception {
          newFileName.close();
          this.close();
       }
+      catch (ClosedChannelException e) {
```

Comment:
in what situation this would happen?

an interrupted exception here means the VM has been interrupted, in what situation the server should be avoided from going down?